src/niscope/system_tests/test_system_niscope.py

```diff
@@ -151,6 +151,8 @@ def test_configure_chan_characteristics(session):
     assert 50.0 == session.input_impedance
 
 
+'''
+# TODO(frank): re-add after issue #650 is fixed.
 # TODO(injaleea): check after issue #639 fixed, will have to modify after according to fix for issue#614
```

Comment:
You can update or remove this TODO. #639 is closed. #614 will break this test so a TODO is optional IMO.

Reply:
Fixed